.zenodo.json

```diff
@@ -1,126 +1,258 @@
 {
-    "creators": [
-      {
-        "affiliation": "HZDR, TU Dresden",
-        "name": "Huebl, Axel",
-        "orcid": "0000-0003-1943-7141"
-      },
-      {
-        "affiliation": "HZDR",
-        "name": "Widera, René",
-        "orcid": "0000-0003-1642-0459"
-      },
-      {
-        "affiliation": "LogMeIn, Inc.",
-        "name": "Worpitz, Benjamin"
-      },
-      {
-        "affiliation": "HZDR, TU Dresden",
-        "name": "Pausch, Richard",
-        "orcid": "0000-0001-7990-9564"
-      },
-      {
-        "affiliation": "HZDR, TU Dresden",
-        "name": "Burau, Heiko"
-      },
-      {
-        "affiliation": "HZDR, TU Dresden",
-        "name": "Garten, Marco",
-        "orcid": "0000-0001-6994-2475"
-      },
-      {
-        "affiliation": "HZDR",
-        "name": "Starke, Sebastian"
-      },
-      {
-        "affiliation": "HZDR, TU Dresden",
-        "name": "Grund, Alexander",
-        "orcid": "0000-0002-7196-8452"
-      },
-      {
-        "affiliation": "HZDR",
-        "name": "Debus, Alexander",
-        "orcid": "0000-0002-3844-3697"
-      },
-      {
-        "affiliation": "HZDR, TU Dresden",
-        "name": "Matthes, Alexander",
-        "orcid": "0000-0002-6702-2015"
-      },
-      {
-        "affiliation": "HZDR",
-        "name": "Bastrakov, Sergei",
-        "orcid": "0000-0003-3396-6154"
-      },
-      {
-        "affiliation": "HZDR",
-        "name": "Steiniger, Klaus",
-        "orcid": "0000-0001-8965-1149"
-      },
-      {
-        "affiliation": "HZDR",
-        "name": "Göthel, Ilja"
-      },
-      {
-        "affiliation": "HZDR, TU Dresden",
-        "name": "Rudat, Sophie"
+  "creators":[
+    {
+      "affiliation":"HZDR",
+      "name":"Bastrakov, Kseniia"
```

Comment:
Bastrakova, it's a russian surname

Reply:
If it's alphabetical list, probably should be also swapped with me ?

Reply:
Yes discussed with @klaus that even if the alphabetic order would not fit, a core developer should be the first author.

Reply:
Yes, I agree. You used the mattermost ping and so pinged the wrong person :)

Reply:
> Yes, I agree. You used the mattermost ping and so pinged the wrong person :)


You are right `@klaus` sry for pinging you.